bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml

```diff
@@ -395,6 +395,12 @@ limitations under the License.
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4j.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I'm a bit confused why this had to change. Can you add a comment?

Reply:
The deps build was failing as we were still pulling in some older versions of slf4j: https://source.cloud.google.com/results/invocations/12b8712b-605a-4315-86c3-0c50dadbd210/targets/cloud-devrel%2Fclient-libraries%2Fjava%2Fjava-bigtable-hbase%2Fpresubmit%2Fdependencies/log. Because this is a test module, I figure the easiest solution was to declare it explicitly. Will add a comment

Reply:
It seems like hbase-shaded-client uses slf4j 1.7.25 and since we are trying to be compatible with hbase, we should use that version

![4oVUA2zfQRQU7oB](https://user-images.githubusercontent.com/23526194/96624612-fbb44d00-12da-11eb-8318-3a017ddbfc14.png)

Reply:
ok, fair enough. I'll close this